Description: Fixed isspace implicit declaration warning.
Author: Eugene Seliverstov <theirix@gmail.com>
Last-Update: 2012-10-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/asn1c/tests/check-70.c
+++ b/asn1c/tests/check-70.c
@@ -13,6 +13,7 @@
 #include <dirent.h>
 #include <assert.h>
 #include <errno.h>
+#include <ctype.h>
 
 #include <PDU.h>
 
